lib/metadata_json_lint.rb

```diff
@@ -45,6 +41,10 @@ def run
   module_function :run
 
   def parse(metadata)
+    # Small hack to use the module settings as defaults but allow overriding for different rake tasks
```

Comment:
To clarify this a bit further: We have different puppet modules in our control-repo and we want to loop over multiple metadata.json files. this allows setting different options for each of them (in theory).